Assets/dbConn.cs

```diff
@@ -4,32 +4,84 @@
 using Mono.Data.Sqlite;
 using System.Data;
 using System;
+using UnityEngine.UI;
 
 public class dbConn : MonoBehaviour
 {
-    void Start()
+    private string conn;
+
+    private List<HighScore> highScores = new List<HighScore>();
+
+    public GameObject scorePrefab;
+
+    public Transform scoreParent;
+
+    public int topRanks;
+
+    public void Start()
+    {
+        highScores.Clear();
+
+        conn = "URI=file:" + Application.dataPath + "/db/db.db";
+
+        ShowScores();
+    }
+
+    //Update ranking after click Ranking button - needed?
+    /*
+    public void EnterRanking()
+    {
+        highScores.Clear();
+
+        Debug.Log(highScores.Count);
+
+        conn = "URI=file:" + Application.dataPath + "/db/db.db";
+
+        ShowScores();
+    }
+    */

```

Comment:
Do we need this commented out code? If not, then perhaps it's safe to remove it completely.

Reply:
Code removed